omnibus/config/projects/chef.rb

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2012-2014 Chef Software, Inc.
+# Copyright 2012-2016 Chef Software, Inc.
```

Comment:
Can we pretty please do these as a single update rather than littering them in unrelated changes?